spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -126,4 +126,34 @@
       end
     end
   end
+
+  describe 'GET#show' do
+    context 'given a valid survey ID' do
```

Comment:
Should not it be a keyword ID?
```suggestion
    context 'given a valid keyword ID' do
```

Reply:
Fixed in [`94a0f67` (#56)](https://github.com/suho/google-search-ruby/pull/56/commits/94a0f67a4744e5efbb7f23d7b1f076b5e191f3c5)